### PR TITLE
Bio orth

### PR DESCRIPTION
**Pull request by dcolinmorgan**
just a small functional coupling via ortholog species demo : database has a dozen+ species with various edge makeups so nice for umap-ing and tough to visualize traditionally

### CODE REVIEW

File: src/python/views/demo_bio_01_funcoup/__init__.py [hunk 1]
```diff
@@ -0,0 +1,307 @@
...
+
+############################################
+#
+#   DASHBOARD SETTINGS
+#
+############################################
+#  Controls how entrypoint.py picks it up
+
+
+app_id = 'app_00'
```
Comment (lmeyerov) [inline src/python/views/demo_bio_01_funcoup/__init__.py:16]: can you change to a later demo ID so doesn't take precedence over the intro demo series?

File: src/python/views/demo_bio_01_funcoup/__init__.py [hunk 2]
```diff
@@ -0,0 +1,307 @@
...
+
+    st.sidebar.title("Choose Edge Weight")
+    edge_to_label = {'PFC':'PFC', 'FBS_max':'FBS_max', 'FBS_PPI':'FBS_PPI',
+       'FBS_Metabolic':'FBS_Metabolic', 'FBS_Signaling':'FBS_Signaling', 'LLR_MEX':'LLR_MEX', 'LLR_PIN':'LLR_PIN',
+       'LLR_SCL':'LLR_SCL', 'LLR_PHP':'LLR_PHP', 'LLR_MIR':'LLR_MIR', 'LLR_TFB':'LLR_TFB', 'LLR_PEX':'LLR_PEX',
+       'LLR_GIN':'LLR_GIN', 'LLR_DOM':'LLR_DOM', 'LLR_QMS':'LLR_QMS', 'LLR_HSA':'LLR_HSA', 'LLR_MMU':'LLR_MMU',
+       'LLR_RNO':'LLR_RNO', 'LLR_CFA':'LLR_CFA', 'LLR_GGA':'LLR_GGA', 'LLR_DRE':'LLR_DRE', 'LLR_CIN':'LLR_CIN',
+       'LLR_DME':'LLR_DME', 'LLR_CEL':'LLR_CEL', 'LLR_BTA':'LLR_BTA', 'LLR_SCE':'LLR_SCE', 'LLR_ATH':'LLR_ATH',
+       'LLR_SPO':'LLR_SPO', 'LLR_BSU':'LLR_BSU', 'LLR_ECO':'LLR_ECO', 'LLR_PFA':'LLR_PFA', 'LLR_DDI':'LLR_DDI',
+       'LLR_SSO':'LLR_SSO', 'LLR_SSC':'LLR_SSC', 'LLR_MJA':'LLR_MJA', 'LLR_OSA':'LLR_OSA','Max_category':'Max_category'
```
Comment (lmeyerov) [inline src/python/views/demo_bio_01_funcoup/__init__.py:116]: can these sorts of menus be dynamically populated by crunching over CSV fields?

File: src/python/views/demo_bio_01_funcoup/__init__.py [hunk 3]
```diff
@@ -0,0 +1,307 @@
...
+
+############################################
+#
+#   SIDEBAR RENDER AERA
+#
+############################################
+# Given URL params, render left sidebar form and return combined filter settings
+
+# #https://docs.streamlit.io/en/stable/api.html#display-interactive-widgets
+def sidebar_area():
```
Comment (lmeyerov) [inline src/python/views/demo_bio_01_funcoup/__init__.py:54]: A bit more description in the side bar and main area on what is being shown may help... what/why wrt topic + how it is rendered, as may not be obvious to everyone

File: src/python/views/demo_bio_01_funcoup/__init__.py (hunk 2 above)
Comment (dcolinmorgan) [reply src/python/views/demo_bio_01_funcoup/__init__.py:116]: just made the overall interface more logical so no need for these fields anymore

File: src/python/requirements-app.txt [hunk 4]
```diff
@@ -1,3 +1,5 @@
 ###
 ### Use this file for your custom view requirements here vs in the requirements-system.txt
-###
\ No newline at end of file
+###
+umap-learn
+dirty_cat
```
Comment (lmeyerov) [inline src/python/requirements-app.txt:5]: Can we remove these? We were thinking of landing as part of this week's release and I saw this is lingering

**Review by lmeyerov — APPROVED** [review]

**Pull request merged**